FreeRTOS/FreeRTOS
# Rename FreeRTOS-IoT-Libraries directory

Add `LTS-Beta1` suffix to the existing `FreeRTOS-IoT-Libraries` directories to support differentiating them from C-SDK development branch libraries that will be added later

Description
-----------
Rename the following 2 directories as:
* `FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries` -> `FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta1`
* `FreeRTOS-Plus/Demo/FreeRTOS_IoT_Libraries` -> `FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1`

Test Steps
-----------
Using custom jobs to verify that demos are building



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
